src/package-install-scripts.js

```diff
@@ -34,6 +34,10 @@ export default class PackageInstallScripts {
   force: boolean;
   artifacts: InstallArtifacts;
 
+  setForce(force: boolean) {
```

Comment:
I think a setter is fine, it's how we already do it (`setArtifacts`), and I have the feeling it makes the code a bit safer (because JS doesn't (yet) have private variables, setters are one of the only ways to clearly state that a variable might be updated from the outside).